README.md

```diff
@@ -7,7 +7,7 @@ It may be used both by mining pools and exchanges.
 
 ## Requirements
 
-* [Node.js](https://nodejs.org/): We recommend and test with version >= 14.0.0, but version >= 10.0.0 will also probably work. Do not use versions < 10.0.0, they are known to have issues.
+* [Node.js](https://nodejs.org/): We recommend and test with version >= 20.0.0.
```

Comment:
suggestion: People won't be able to use any lower version anyway, since we have `engine-strict=true` in the `.npmrc` file.

Should we change this sentence to reflect this?

Reply:
Done on [eca6313](https://github.com/HathorNetwork/hathor-wallet-headless/pull/405/commits/eca6313bace5fed042cf70a10ffbc2dc571e1224)